pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -85,11 +85,11 @@ func (p *ProjectService) workloadType() string {
 }
 
 // serviceType returns service type for project service workload
-func (p *ProjectService) serviceType() (string, error) {
+func (p *ProjectService) serviceType() (config.ServiceType, error) {
 	serviceType := p.SvcK8sConfig.Service.Type
 
 	// @step validate whether service type is set properly when node port is specified
-	if !strings.EqualFold(serviceType, string(v1.ServiceTypeNodePort)) && p.nodePort() != 0 {
+	if !config.ServiceTypesEqual(serviceType, config.NodePortService) && p.nodePort() != 0 {
```

Comment:
minor comment, `v1.ServiceTypeNodePort` is unlikely to change in contrast to the value of `config.NodePortService`... I know they both represent the same thing but perhaps we should stick to the former?